src/main/java/com/courtcanva/ccfranchise/mappers/OrderMapper.java

```diff
@@ -1,15 +1,42 @@
 package com.courtcanva.ccfranchise.mappers;
 
 import com.courtcanva.ccfranchise.dtos.orders.OrderGetDto;
+import com.courtcanva.ccfranchise.dtos.orders.OrderOpenPostDto;
 import com.courtcanva.ccfranchise.dtos.orders.OrderPostDto;
+import com.courtcanva.ccfranchise.models.DesignInformation;
 import com.courtcanva.ccfranchise.models.Order;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 import org.mapstruct.ReportingPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public interface OrderMapper {
+
+    Logger log = LoggerFactory.getLogger(OrderMapper.class);
```

Comment:
You can use @Slf4j instead of this

Reply:
@slf4j doesn't work in here( Interface).

I used workround according to [stackoverflow](https://stackoverflow.com/a/64404268/14777101)